qualtran/bloqs/mod_arithmetic/mod_multiplication.py

```diff
@@ -265,3 +273,463 @@ def _modmul_symb() -> CModMulK:
 
 
 _C_MOD_MUL_K_DOC = BloqDocSpec(bloq_cls=CModMulK, examples=(_modmul_symb, _modmul))
+
+
+@frozen
+class SingleWindowModMul(Bloq):
+    r"""Performs modular multiplication on a single windowed.
+
+    Applies
+    $$
+        \ket{x}\key{y}\key{t}\ket{0} \rightarrow \ket{x}\key{y}\ket{t+xy \mod p} \ket{xy \mod 2^w}
+    $$
+
+    Where:
+
+    - $w$ is the window size.
+    - $p$ is the modulus.
+
+    Args:
+        bitsize: size of the numbers.
+        window_size: size of the window.
```

Comment:
please match the order of the args doc with the order of the init args

Reply:
done